### ISSUE TITLE
[MENFORCER-254] version 1.4.1 cause extra-enforcer-rules ITs failures

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-enforcer#745]
**[Hervé Boutemy](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=hboutemy)** opened **[MENFORCER-254](https://issues.apache.org/jira/browse/MENFORCER-254?redirect=false)** and commented

when building extra-enforcer-rules http://www.mojohaus.org/extra-enforcer-rules/ with 1.4.1, some ITs fail: no issue with 1.4 or any other release

---

**Affects:** 1.4.1

**Attachments:**
- [pom.xml](https://issues.apache.org/jira/secure/attachment/12806901/pom.xml) (_2.45 kB_)

**Issue Links:**
- [MPOM-127](https://issues.apache.org/jira/browse/MPOM-127) upgrade extra-enforcer-rules to 1.0-beta-4

**Comment by jira-importer** [comment]
**[Karl Heinz Marbaise](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=khmarbaise)** commented

The list of failures:

```
[INFO] -------------------------------------------------
[INFO] Build Summary:
[INFO]   Passed: 23, Failed: 5, Errors: 0, Skipped: 0
[INFO] -------------------------------------------------
[ERROR] The following builds failed:
[ERROR] *  mojo-1682/pom.xml
[ERROR] *  mojo-1731/pom.xml
[ERROR] *  mojo-1744/pom.xml
[ERROR] *  mojo-1769/pom.xml
[ERROR] *  smokes/pom.xml
[INFO] -------------------------------------------------
```

**Comment by jira-importer** [comment]
**[Hervé Boutemy](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=hboutemy)** commented

I don't know if it is strictly the same cause, but I started to find NPE issues with version 1.4.1 while building Maven core, and I extracted the failing build configuration

**Comment by jira-importer** [comment]
**[Karl Heinz Marbaise](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=khmarbaise)** commented

[Hervé Boutemy](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=hboutemy) Hi Hervé, after taking a deeper look into that I have found that the problem is based on the older version of extra-enforcer using 1.0-beta-3 fails but if you use 1.0-beta-4 it works. This [issue has been solved for 1.0-beta-4](https://github.com/mojohaus/extra-enforcer-rules/issues/13).

**Comment by jira-importer** [comment]
**[Hervé Boutemy](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=hboutemy)** commented

ok, thank you: we should have upgraded dependency in maven-parent 30...
